llvm/lib/Target/RISCV/AsmParser/RISCVAsmParser.cpp

```diff
@@ -1210,7 +1210,7 @@ struct RISCVOperand final : public MCParsedAsmOperand {
   }
 
   void addRegRegOperands(MCInst &Inst, unsigned N) const {
-    assert(N == 1 && "Invalid number of operands!");
+    assert(N == 2 && "Invalid number of operands!");
```

Comment:
How does this relate to the rest of the patch? A .ll test shouldn't use the AsmParser.

Reply:
I think it's because I'm using `let MIOperandInfo = (ops GPR:$base, GPR:$offset)` for CVrr operand, so it triggers the assert.